sorc/link_workflow.sh

```diff
@@ -395,7 +398,7 @@ cd "${script_dir}"   ||   exit 8
       webtitle.fd
       do
         if [[ -d "${prog}" ]]; then rm -rf "${prog}"; fi
-        ${LINK} gfs_utils.fd/src/${prog} .
+        ${LINK} gfs_utils.fd/src/"${prog}" .
```

Comment:
Quote whole path

Reply:
Changed to: `${LINK} "gfs_utils.fd/src/${prog}" .`